src/app.module.ts

```diff
@@ -1,18 +1,35 @@
 import { Module } from '@nestjs/common';
 import { TypeOrmModule } from '@nestjs/typeorm'
 import { ConfigModule } from '@nestjs/config'
+import { Connection } from 'typeorm';
 
 import ormconfig from './config/ormconfig';
 import { HelloWorldController } from './controllers/hello-world.controller';
 
+import { SeedsModule } from './modules/seeds/seeds.module'
+
+import { SeedsService } from '@services/seeds.service';
+
 @Module({
   imports: [
     ConfigModule.forRoot({
-      envFilePath: [`.env.${process.env.NODE_ENV}`, '.env']
+      envFilePath: ['../env']
```

Comment:
In this case, we could just do the following right? That way we can keep the `.env` variables inside the `.env.development` file? 🤔 

```suggestion
      envFilePath: [`../.env.${process.env.NODE_ENV}`, '../.env']
```